js/about/bookmarks.js

```diff
@@ -180,14 +180,28 @@ class BookmarksList extends ImmutableComponent {
   }
 }
 
+class SearchResults extends React.Component {
+  render () {
+    return (
+      <list className='bookmarkList'>
+      {
+        this.props.bookmarks.map((bookmark) => <BookmarkItem bookmark={bookmark} />)
+      }
+      </list>
+    )
+  }
+}
+
 class AboutBookmarks extends React.Component {
   constructor () {
     super()
     this.onChangeSelectedFolder = this.onChangeSelectedFolder.bind(this)
```

Comment:
You could not clear but just search only within the selected folder and descendants would be fine too.  If you do the search all though maybe we should show a selected virtual item like chrome does for search terms so there's no ambiguity.